PS 2 Elshana

```diff
@@ -0,0 +1,183 @@
+# 2) Make sure to comment wherever it is needed
+# 3) Every student should submit his\hers own copy
+
+
+##############################################################################
+# Problem 1 (70%)
+##############################################################################
+
+# You'll be working with adult.data . Here's a brief description of the 
+# variables:
+# age: continuous.
+# workclass: Private, Self-emp-not-inc, Self-emp-inc, Federal-gov, Local-gov, State-gov, 
+#            Without-pay, Never-worked.
+# fnlwgt: continuous.
+# education: Bachelors, Some-college, 11th, HS-grad, Prof-school, Assoc-acdm, Assoc-voc,
+#            9th, 7th-8th, 12th, Masters, 1st-4th, 10th, Doctorate, 5th-6th, Preschool.
+# education-num: continuous.
+# marital-status: Married-civ-spouse, Divorced, Never-married, Separated, Widowed, Married-spouse-absent, Married-AF-spouse.
+# occupation: Tech-support, Craft-repair, Other-service, Sales, Exec-managerial, Prof-specialty, Handlers-cleaners, Machine-op-inspct, Adm-clerical, Farming-fishing, Transport-moving, Priv-house-serv, Protective-serv, Armed-Forces.
+# relationship: Wife, Own-child, Husband, Not-in-family, Other-relative, Unmarried.
+# race: White, Asian-Pac-Islander, Amer-Indian-Eskimo, Other, Black.
+# sex: Female, Male.
+# capital-gain: continuous.
+# capital-loss: continuous.
+# hours-per-week: continuous.
+# native-country: United-States, Cambodia, England, Puerto-Rico, Canada, Germany, Outlying-US(Guam-USVI-etc), India, Japan, Greece, South, China, Cuba, Iran, Honduras, Philippines, Italy, Poland, Jamaica, Vietnam, Mexico, Portugal, Ireland, France, Dominican-Republic, Laos, Ecuador, Taiwan, Haiti, Columbia, Hungary, Guatemala, Nicaragua, Scotland, Thailand, Yugoslavia, El-Salvador, Trinadad&Tobago, Peru, Hong, Holand-Netherlands.
+# salary: >50K,<=50K
+
+# Further description is available in adult.names file
+
+# 1) read the data, make sure columns are read in the right format, i.e. character, double, etc.
+library(readr)
+adult <- read_csv("C:/Users/user/Documents/GitHub/RfEX_stud_version/ps2/adult.data", col_names = FALSE)
+View(adult)
+library(magrittr)
+
+library(tidyverse)
+
+names(adult)[4]<-paste("Education")
+names(adult)[5]<-paste("educ.numbers")
+names(adult)[9]<-paste("race")
+names(adult)[10]<-paste("sex")
+names(adult)[1]<-paste("Age")
+names(adult)[12]<-paste("0")
+names(adult)[8]<-paste("family.condition")
+names(adult)[15]<-paste("salary")
+names(adult)[2]<-paste("workclass")
+names(adult)[6]<-paste("maritalstatus")
+names(adult)[7]<-paste("occupation")
+names(adult)[8]<-paste("relationship")
+names(adult)[11]<-paste("capitalgain")
+names(adult)[12]<-paste("capitalloss")
+names(adult)[13]<-paste("hoursperweek")
+names(adult)[14]<-paste("nativecontry")
+names(adult)[3]<-paste("fnlwgt")
+attach(adult)
+
+# 2) How many men and women are represented in the dataset? 
+
+a <- sum(adult$sex == "Female")
+a
+b <- sum(adult$sex == "Male")
+b
```

Comment:
names of the variable should be representative 

a and b doesn't mean anything to be, but num_men and num_women does.
+ using the summarize you would only need a single line to compute everything